### PR TITLE
Handle pagination on subscan transactions

### PR DESCRIPTION
**Pull request by maycon-mello**

### CODE REVIEW

File: src/features/transactions/transactions-slice.js [hunk 1]
```diff
@@ -55,45 +56,63 @@ export const transactionsSelectors = {
 };
 
 export const transactionsOperations = {
+  loadExternalTransactions: account => async (dispatch, getState) => {
+    const realm = getRealm();
+    const dbTransactions = realm.objects('Transaction').toJSON();
```
[the remark this answers is not in the document]
Comment (mike-parkhill) [reply src/features/transactions/transactions-slice.js:61]: yeah, blindly loading everything into memory could be dangerous. Paging it probably makes sense.

[the remark this answers is not in the document]
Comment (mike-parkhill) [reply src/features/transactions/transactions-slice.js:61]: agreed offline to make paging part of the SDK changes rather doing it twice.

File: src/core/subscan.js [hunk 2]
```diff
@@ -20,9 +20,5 @@ export function fetchTransactions({address, page = 0, row = 50}) {
         ...res.data,
         hasNextPage: (page + 1) * row < res.data.count,
       };
-    })
-    .catch(err => {
```
Comment (mike-parkhill) [inline src/core/subscan.js]: we're good to let the error escape?

Comment (maycon-mello) [reply src/core/subscan.js]: Actually I removed this in the latest commit I think

Comment (maycon-mello) [reply src/core/subscan.js]: We should propagate the error

Comment (mike-parkhill) [reply src/core/subscan.js]: yeah, the catch is gone and I was just making sure that allowing the error to bubble out now wasn't going to cause other problems.

**Review by mike-parkhill — APPROVED** [review]

**Pull request merged**